src/ui/components/Rating/index.js

```diff
@@ -92,12 +93,13 @@ export class RatingBase extends React.Component {
         `styleSize=${styleSize} is not a valid value; ` +
         `possible values: ${RATING_STYLE_SIZES.join(', ')}`);
     }
-    let description;
-    if (rating) {
+    let description = 'Click to rate this add-on';
```

Comment:
Any strings that we show to users need to be wrapped in a gettext call; this string will not get localised for users which would be bad.

That said: why is this being set by default? Since we could be setting a `title` attribute each rating star shouldn't we skip the title on the entire rating wrapper? I don't think this is a good message anyway, it can probably just go.